io.github.yairm210.unciv.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.yairm210.unciv</id>
+  <name>UnCiv</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+  <developer_name>Yair Morgenstern</developer_name>
+  <summary>Turn-based strategy game</summary>
+  <summary xml:lang="fr">Jeu de stratégie au tour par tour</summary>
+  <description>
+    <p>A reimplementation of the most famous civilization-building game ever—fast, small, no ads, free forever!</p>
+    <p>Build your civilization, research technologies, expand your cities and defeat your foes!</p>
+    <p>Requests? Bugs? Visit the todo list at https://github.com/yairm210/UnCiv/issues, every small help is welcome!</p>
+    <p>Questions? Comments? Just bored? Join us on https://discord.gg/bjrB4Xw</p>
+    <p>Want to help translating the game into your language? Send me an email to yairm210@hotmail.com.</p>
+    <p>Grok Java, Kotlin or C#? Join us at https://github.com/yairm210/UnCiv.</p>
```

Comment:
Those should go into the a url tag. Please see the specs https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Consider it done (except for questions, didn't find a suitable link type for a realtime chat). However, I would have liked not to change too much the maintainer's wording/description for this initial release, so can this stay here as well?

Reply:
It would be better to get it right before merging.